internal/pkg/flink/internal/store/store_utils.go

```diff
@@ -233,43 +263,52 @@ func parseResetStatement(statement string) (string, error) {
 	}
 
 	if len(words) > 2 {
-		return "", &types.StatementError{Msg: "Error: too many keys for RESET provided. Usage example: RESET 'key'."}
+		return "", &types.StatementError{
+			Msg:   "too many keys for RESET provided",
+			Usage: []string{"RESET 'key'"},
+		}
 	}
 
 	isFirstWordReset := strings.ToUpper(words[0]) == config.ConfigOpReset
 	key := strings.ToLower(words[1])
 	if !isFirstWordReset {
-		return "", &types.StatementError{Msg: "Error: Invalid syntax for RESET. Usage example: RESET 'key'."}
+		return "", &types.StatementError{
+			Msg:   "Invalid syntax for RESET",
```

Comment:
Your error messages are inconsistent. Some start with an uppercase letter and some don't. My vote is for starting them with a lowercase letter! https://staticcheck.io/docs/checks#ST1005
```suggestion
			Msg:   "invalid syntax for RESET",
```

Reply:
Made all of them lowercase and removed punctuation!